$NetBSD: patch-configure,v 1.2 2014/01/24 11:29:30 obache Exp $

* rbconfig_vendorarchhdr dir may not be defined even if rbconfig_hdrdir exists

--- configure.orig	2014-01-23 02:36:54.000000000 +0000
+++ configure
@@ -13989,6 +13989,10 @@ if test "$ruby_available" = "yes"; then
     fi
     if test -d "$rbconfig_vendorarchhdrdir"; then
       RUBY_CFLAGS="$RUBY_CFLAGS -I$rbconfig_vendorarchhdrdir"
+    else
+      if test "$rbconfig_vendorarchdir" != ""; then
+        RUBY_CFLAGS="$RUBY_CFLAGS -I$rbconfig_vendorarchdir"
+      fi
     fi
     if test -d "$rbconfig_rubyarchhdrdir"; then
       RUBY_CFLAGS="$RUBY_CFLAGS -I$rbconfig_rubyarchhdrdir"
